eduNEXT/edunext-platform
# HA/EDNX/FEC-24 - Web certificates improvements.

## Description: 

This PR adds support to show the certificates tab in instructor tab in lms to an instructor user.
Additional, add general improvements to generate certificates in the fastest way.

## Test:

 - Be able to generate certificates for any course.
 - Create a user with instrcutor rol.
 - Go to lms and log in.
 - Go into course outline.
 - Go to instructor tab.
 - Finally, the certificates tab should be displayed.

## Reviewers:
